frontend/test/metabase/lib/expressions/fuzz.recursive-parser.unit.spec.js

```diff
@@ -1,8 +1,9 @@
+import _ from "underscore";
 import { parse } from "metabase-lib/expressions/recursive-parser";
 
 import { generateExpression } from "./generator";
 
-const fuzz = process.env.MB_FUZZ ? describe : describe.skip;
+const fuzz = process.env.MB_FUZZ ? describe : _.noop;
```

Comment:
nice! does this get rid of 70k skipped tests?

Reply:
yup